src/octokit/get-octokit-throttle-options.ts

```diff
@@ -1,11 +1,11 @@
 import Bottleneck from "bottleneck";
-import Redis from "ioredis";
+import { Redis, RedisOptions } from "ioredis";
```

Comment:
```suggestion
import { Redis, type RedisOptions } from "ioredis";
```

small mistake from  my side.